custom-vu/src/main/kotlin/jces1209/vu/page/boards/cloud/BoardPage.kt

```diff
@@ -0,0 +1,15 @@
+package jces1209.vu.page.boards.cloud
+
+import java.net.URI
+
+interface BoardPage {
+
+    val uri: URI
+    fun waitForBoardPageToLoad(): BoardContent
+    fun areThereIssues(): Boolean
+
+    /**
+     * Board must have issues
+     */
+    fun issueBentoView(): BoardPage
```

Comment:
Interesting.
Is this the reason why we had to remove the generic `jces1209.vu.page.boards.BoardPage` in favor of this Cloud-specific one?

I'd try to hide these differences in implementations, rather than exposing them in the interface.
Let's think like a Jira user migrating from Server to Cloud. I have my workflow. I go to the board and I can preview issues without leaving the board. I don't know what Bento is. I don't really care if it's the "new experience" or old. The workflow is the same.

The implementation underneath can detect whether it's Bento or not, we do it already in: https://github.com/atlassian/jces-1209/blob/3c414197e4fed170be0af9b91369f4ea53b1255a/custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt#L29-L35

User-level abstractions are important, they allow us to compare stuff across Cloud and Server. This is the only reason why I'm _requesting changes_ in this PR.

- [ ] avoid tech-specific abstractions in page object interfaces - express the user experience instead

Reply:
Thanks. I totally agree. just thought bento is a short name for the `preview issue` designation.